util/weekly_build/03_SetupEnv.sh

```diff
@@ -12,11 +12,11 @@ set -x
 
 for compiler in $COMPILERS; do
   for template in $TEMPLATES; do
-    envname=build-$template-${compiler/@/-}
+    [[ ${compiler} == *"@="* ]] && envname=build-$template-${compiler/@=/-} || envname=build-$template-${compiler/@/-}
```

Comment:
I don't mind this, but for readability, what about either `$(echo $compiler | perl -pe 's|@=?|-|')` for hyphenizing the compiler spec, or something like
```
envname=build-$template-${compiler/@=/-}
envname=${envname/@/-}
```

Reply:
That's fine with me, but let's keep perl out of this if we can ;-) Would you mind including the update to
```
envname=build-$template-${compiler/@=/-}
envname=${envname/@/-}
```
in your next round of changes?

Reply:
Heh :) Sure